docs/source/gs-processing/usage/emr.rst

```diff
@@ -77,177 +77,93 @@ inline policy:
     ]
     }
 
-Create a security group that will allow us to SSH to the leader instance
-------------------------------------------------------------------------
-
-In order to be able to launch Spark jobs from within the leader instance
-we will need to create an EC2 security group that will allow us to login
-to the EMR leader.
-
-To do so follow the `AWS docs <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/authorizing-access-to-an-instance.html#add-rule-authorize-access>`_,
-and make note of the security group ID, e.g `sg-XXXXXXXXXXXXXXXXX`.
-
-Launch an EMR cluster with the appropriate permissions
-------------------------------------------------------
+Launch an EMR cluster with GSProcessing step
+--------------------------------------------
 
 Once our roles are set up, that is we have an EMR EC2 instance role,
 and a user we can use to launch clusters, we can launch a cluster
-configured to allow us to run jobs with the GSProcessing EMR on EC2
-Docker image. We have tested GSProcessing with EMR 7.0.0 and EMR 6.10.0,
+configured so it will run a GSProcessing job with the GSProcessing EMR on EC2
+Docker image, then terminate. We have tested GSProcessing with EMR 7.0.0 and EMR 6.10.0,
```

Comment:
I am thinking about the design choice here, because sometimes we may launch the gsprocessing jobs on the large clusters which may be tough to set up multiple times because of some problem like quota. May we provide an option for use to pertain/use existing clusters to launch the job? 

Think about a general situation, users provide some wrong input path but they may not be able to launch the clusters again to start the job.

Reply:
Yeah, I was also considering if we want to keep parts from both guides, have the EMR Step one first because it's easier, and also demo how to submit jobs on persistent clusters, which is more or less what we have currently.

Reply:
Maybe we can do that after we merge https://github.com/awslabs/graphstorm/pull/895 @jalencato ?

Reply:
After offline discussion: we can first merge 902 PR, and then 895 PR, and finally add another section in advanced usage.